README.md

```diff
@@ -87,3 +87,20 @@ vim.g.asyncrun_program = vim.empty_dict()
 vim.api.nvim_command("let g:asyncrun_program.vcvars64 = { opts -> '\"C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Auxiliary/Build/vcvars64.bat\" && ' .. opts.cmd }")
 vim.g.cmake_asyncrun_options = vim.tbl_extend('force', vim.g.cmake_asyncrun_options, {program = 'vcvars64'})
 ```
+
+## DAP configuration example
```

Comment:
Can I also ask you to mention that this example is for `CodeLLDB` specially? Because by default I defined the settings for `lldb-vscode` that comes with LLVM 11+.

Reply:
Yes, `CodeLLDB` and `lldb-vscode` are different, the former is server type and the latter is executable type, `lldb-vscode` does not work with `nvim-dap` on windows AFAIK.